src/components/ApplicationsModal/ApplicationsModal.tsx

```diff
@@ -1602,7 +1602,10 @@ export const ApplicationsModal: React.FC<ApplicationsModalProps> = ({
                         <Grid item md={6}>
                           <Button
                             color="primary"
-                            disabled={!hasChanged()}
+                            disabled={
+                              !getValues("appName") || getValues("appRedirectURI") === "https://" ||
```

Comment:
"https://" is used in lines 216, 286, 931 and now 1606. This should be moved into a const.